VisualStudio/winconfig.h

```diff
@@ -1,9 +1,13 @@
-#ifndef __WINCONFIG_H
-#define __WINCONFIG_H
-
-/* Enable schemes supported on Windows */
-#define ENABLE_KEX_NTRU
-#define ENABLE_KEX_RLWE_NEWHOPE
-#define ENABLE_KEX_SIDH_MSR
-#define ENABLE_SIG_PICNIC
-#endif
+#ifndef __WINCONFIG_H
+#define __WINCONFIG_H
+
+/* Enable schemes supported on Windows */
+#define ENABLE_KEX_NTRU
+#define ENABLE_KEX_RLWE_NEWHOPE
+#define ENABLE_KEX_SIDH_MSR
+#define ENABLE_SIG_PICNIC
+#define OQS_ENABLE_KEM_frodokem_640_aes
+#define OQS_ENABLE_KEM_frodokem_976_aes
+#define OQS_ENABLE_KEM_frodokem_640_cshake
+#define OQS_ENABLE_KEM_frodokem_976_cshake
```

Comment:
Do you want to add `OQS_ENABLE_KEM_sike_p503` and `OQS_ENABLE_KEM_sike_p751`?  The `KEX_` versions will be going away in the not-too-distant future.

Reply:
I'll do that as part of issue #350, while I harmonize the SIDH and SIKE code. I just wanted to add at least one KEM with the API to enable the test_kem program to work. All the remaining ones will need to be added one by one, and may take some adjusting to work on Windows; I didn't want to block integrating the KEM API in VS.

Reply:
Understood, thanks.